app/domain/errors.rb

```diff
@@ -338,6 +338,21 @@ module AuthnAzure
         code: "CONJ00054E"
       )
     end
+
+    module AuthnGcp
+
+      InvalidAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token field {0} is invalid. The format should be " \
+              "'conjur/<account_name>/<host_id>'",
+        code: "CONJ00066E"
+      )
+
+      TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+        msg:  "Claim '{0-claim-name}' not found or empty in token. " \
```

Comment:
@shulifink please review

Reply:
fine. What should they do about this? Can we give them some direction?

Reply:
not really. we tell them which claim is missing from the token so they can take a look. in `authn-gcp` we do tell them.